test/test.js

```diff
@@ -0,0 +1,70 @@
+var heroesTest = [
+	{id:1,name:'moses'},
+	{id:2,name:'jody'},
+	{id:3,name:'jany'},
+	{id:4,name:'gon'},
+	{id:5,name:'ishai'}
+];
+const chai = require('chai');
+let assert = require('assert');
+var should = chai.should();
+var chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+sinon = require('sinon')
+var server = require('../server.js');
+let actions = require('../actions');
+actions = new actions.action();
+
+describe('server test', () => {
+
+	describe('GET', function () {
+		it('should get all heroes', function (done) {
+			chai.request(server)
+				.get('/heroes')
+				.end(function (err, res) {
+					res.body[0].should.have.property('id');
```

Comment:
your test is incorrect.
u should test that u get the same number of heros u expect